### PR TITLE
add standard for docstrings

### LINKED ISSUE #10: Docstring format
To help with understanding how functions and classes work, adopt the Google standard for docstrings: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

### PR DESCRIPTION
**Pull request by jdkandersson**
Closes #10

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -195,6 +195,23 @@ unnecessary evaluation of the string formatting. Even if the formatting input
...
 
+## Docstrings
+
+Modules, functions and classes without a well formatted description can be
+difficult to understand and maintain, especially if the function is longer than
+just a few lines.
+
+Each module, function and class should have a docstring. The Google standard on
+docstring applies:
+[docstring guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).
```
Comment (mthaddon) [inline CONTRIBUTING.md:206]: Should we be referencing https://peps.python.org/pep-0257/ instead/as well as this?

Comment (merkata) [reply CONTRIBUTING.md:206]: Note that the paragraph on docstrings from the Google styleguide mentions PEP257, so we might only clarify that the guide steps on a python standard for docstrings and potentially include the link as well (although you can click it from the Google styleguide as well).

**Review by merkata — DISMISSED** [review]
Looks good, nice.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jdkandersson) [reply CONTRIBUTING.md]: I have added PEP257 in as well, the Google standard is a little more specific and provides lots of examples which makes docstrings even easier to read

**Review by merkata — APPROVED** [review]
Looks good.

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Comment by varshigupta12** [comment]
LGTM :)

**Pull request merged**